frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -136,18 +140,27 @@ export const supportLogic = kea<supportLogicType>([
             const email = userLogic.values.user?.email
 
             const zendesk_ticket_uuid = uuid()
+            const subject =
+                SUPPORT_KIND_TO_SUBJECT[kind ?? 'support'] +
+                ': ' +
+                (target_area ? TARGET_AREA_TO_NAME[target_area] : 'General') +
+                ' (' +
+                zendesk_ticket_uuid +
+                ')'
             const payload = {
                 request: {
                     requester: { name: name, email: email },
-                    subject: 'Help in-app',
+                    subject: subject,
                     comment: {
                         body:
```

Comment:
We can only include formatted links by using HTML. In that case we'd send `html_body` instead of `body`.
https://developer.zendesk.com/api-reference/ticketing/tickets/ticket-requests/#create-request